.github/workflows/ubuntu-ci-x86_64-gnu.yaml

```diff
@@ -53,88 +53,99 @@ jobs:
-          spack config add "packages:all:providers:mpi:[openmpi@5.0.3]"
-          spack config add "packages:all:compiler:[gcc@11.4.0]"
-          sed -i "s/\['\%aocc', '\%apple-clang', '\%gcc', '\%intel', '\%oneapi'\]/\['\%gcc'\]/g" $ENVDIR/spack.yaml
-
-          # Add additional variants for MET packages, different from config/common/packages.yaml
-          spack config add "packages:met:variants:+python +grib2 +graphics +lidar2nc +modis"
-
-          # Concretize and check for duplicates
-          spack concretize 2>&1 | tee log.concretize.gnu-11.4.0-buildcache
-          ${SPACK_STACK_DIR}/util/show_duplicate_packages.py -d log.concretize.gnu-11.4.0-buildcache -i fms -i crtm -i esmf -i mapl
-
-          # Add and update source cache
-          spack mirror add local-source file:///home/ubuntu/spack-stack/source-cache/
-          spack mirror create -a -d /home/ubuntu/spack-stack/source-cache/
-
-          # Add binary cache if requested
-          if [ "$USE_BINARY_CACHE" = true ] ; then
-            set +e
-            spack mirror add local-binary file:///home/ubuntu/spack-stack/build-cache/
-            spack buildcache update-index local-binary || (echo "No valid binary cache found, proceed without" && spack mirror rm local-binary)
-            set +e
-            echo "Packages in spack binary cache:"
-            spack buildcache list
-          fi
-
-          # Break installation up in pieces and create build caches in between
-          # This allows us to "spin up" builds that altogether take longer than
-          # six hours, and/or fail later in the build process.
-
-          # base-env
-          echo "base-env ..."
-          spack install --fail-fast --source --no-check-signature base-env 2>&1 | tee log.install.gnu-11.4.0-buildcache.base-env
-          spack buildcache create -a -u /home/ubuntu/spack-stack/build-cache/ base-env
-
-          # jedi-base-env
-          echo "jedi-base-env ..."
-          spack install --fail-fast --source --no-check-signature jedi-base-env 2>&1 | tee log.install.gnu-11.4.0-buildcache.jedi-base-env
-          spack buildcache create -a -u /home/ubuntu/spack-stack/build-cache/ jedi-base-env
-
-          # the rest
-          echo "unified-env ..."
-          spack install --fail-fast --source --no-check-signature 2>&1 | tee log.install.gnu-11.4.0-buildcache.unified-env
-          spack buildcache create -a -u /home/ubuntu/spack-stack/build-cache/
-
-          # Remove binary cache for next round of concretization
-          if [ "$USE_BINARY_CACHE" = true ] ; then
-            spack mirror rm local-binary
-          fi
-
-          # Remove buildcache config settings
-          spack config remove config:install_tree:padded_length
 
-          # Next steps: synchronize source and build cache to a central/combined mirror?
-
-          # Cleanup
-          spack clean -a
-          spack env deactivate
+          declare -a TEMPLATES=("unified-dev" "skylab-dev")
```

Comment:
There are no code changes between lines 66 and 146 - they are the same as lines 56 to 137 on the left side. The only diff is the indentation for the loop over templates.